UnityProject/Assets/Scripts/UI/AdminTools/GUI_AdminTools.cs

```diff
@@ -17,6 +17,7 @@ public class GUI_AdminTools : MonoBehaviour
 		[SerializeField] private GameObject playerChatPage = null;
 		[SerializeField] private GameObject playersScrollView = null;
 		[SerializeField] private GameObject CentCommPage = null;
+		[SerializeField] private GameObject EventsManagerPage = null;
```

Comment:
I would suggest changing name to eventsManagerPage because it is a private variable

Reply:
fixed